models.py

```diff
@@ -322,6 +322,13 @@ def __eq__(self, other):
                 return False
         return True
 
+    def share_uni(self, other):
```

Comment:
```suggestion
    def shares_uni(self, other):
```

Reply:
This EVC shares a UNI with the other?

Reply:
To me, 'shares_uni' or 'shares_uni_with(other)' looks more like a question (which is the right thing, since it returns a boolean) than 'share_uni', which can confused with a command to 'share' it.